great-idea/css/services.css

```diff
@@ -0,0 +1,271 @@
+/*  http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/*  HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+	margin: 0 auto;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+	
+	margin-bottom: 15px;
+	font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+}
+
+/* Your code starts here */
+
+/* ************************************************
+*                    CONTAINER                    *
+***************************************************/ 
+.container {
+	max-width: 880px;
+	margin: 0 auto;
+}
+
+/* ************************************************
+*                       NAV                       *
+***************************************************/ 
+nav {
+	display: flex;
+	justify-content: space-between;
+    align-items: center;
+
+    margin-top: 28px;
+    margin-bottom: 20px;
+}
+
+nav > a {
+	font-size: 14px;
+	color: #B3B3B3;
+	text-decoration: none;
+}
+
+/* ************************************************
+*                  SERVICES HEADER                *
+***************************************************/ 
+.service-header-container {
+    border-bottom: 1px solid black;
+}
+
+.services-header {
+    padding: 30px 20px 50px 20px;
```

Comment:
Very minor thing, but the padding on the services header seems to be inconsistent with the rest of the page.

Reply:
Thank you for pointing this out: This was a design choice. Not sure if it was good one though. 

The idea was that if the window size was exactly the width of the page container I wanted a little padding for the services header so it wasn't right up against the edge of the window. The padding I applied to the services-header aligns _its_ text with the text for all of the services-sections. 

However, on second glance I notice that the services-info section at the bottom does not have this padding applied. So, I did _not_ apply this padding consistently throughout the page. 

Whether applying the padding at all was a good design choice, I'm not sure. But I do agree that it should be consistent with the entire page.